modules/_labs/select/react/stories/stories_VisualTesting.tsx

```diff
@@ -0,0 +1,190 @@
+/// <reference path="../../../../../typings.d.ts" />
+import * as React from 'react';
+import {storiesOf} from '@storybook/react';
+import withReadme from 'storybook-readme/with-readme';
+import {StaticStates} from '@workday/canvas-kit-labs-react-core';
+import {
+  ComponentStatesTable,
+  permutateProps,
+  customColorTheme,
+} from '../../../../../utils/storybook';
+import Select from '../lib/Select';
+import SelectBase from '../lib/SelectBase';
+import SelectOption from '../lib/SelectOption';
+import README from '../README.md';
+
+const options = [
+  {label: 'E-mail', value: 'email'},
+  {label: 'Phone', value: 'phone'},
+  {label: 'Fax (disabled)', value: 'fax', disabled: true},
+  {label: 'Mail', value: 'mail'},
+  {label: 'Mobile Phone', value: 'mobile_phone'},
+  {
+    label: 'The Ontologically Anthropocentric Sensory Immersive Simulation',
+    value: 'oasis',
+  },
+];
+
+const normalizedOptions = options.map(option => {
+  return {
+    data: {},
+    disabled: option.disabled || false,
+    id: option.value,
+    label: option.label || option.value,
+    value: option.value,
+  };
+});
+
+storiesOf('Labs|Select/React/Visual Testing', module)
```

Comment:
Should i change this to storybook csf?

Reply:
Saving CSF for later after visual testing stories have stabilized a bit from changes in other PRs (#606 and #641 ).